src/components/AddAnnonce/Description/index.jsx

```diff
@@ -0,0 +1,126 @@
+import { useState } from 'react';
+import Proptypes from 'prop-types';
+import SDescription from './style';
+
+const Description = ({
+  setDistrict,
+  setCity,
+  setDescribe,
+  setRent,
+  inputDistrict,
+  inputCity,
+  inputDescribe,
+  inputRent,
+}) => {
+  const [isLille, setIsLille] = useState(false);
+
+  const handleChangeDistrict = (evt) => {
+    setDistrict(evt.target.value);
+  };
+  const handleChangeCity = (evt) => {
+    setCity(evt.target.value);
+    if (evt.target.value === 'Lille') {
+      setIsLille(true);
+    } else {
+      setIsLille(false);
+    }
+  };
+  const handleChangeDescribe = (evt) => {
+    setDescribe(evt.target.value);
+  };
+  const handleChangeRent = (evt) => {
+    setRent(evt.target.value);
+  };
+
+  return (
+    <SDescription>
+      <div className="partContainer">
+        <div className="addressCity">
+          <div className="city">
+            <h2>Ville :</h2>
+            <select type="text" value={inputCity} onChange={handleChangeCity}>
+              <option selected value="">
+                {' '}
+              </option>
+              <option value="Lille">Lille</option>
+              <option value="La Madeleine">La Madeleine</option>
+              <option value="Lambersart">Lambersart</option>
+              <option value="St André">St André</option>
+              <option value="Lomme">Lomme</option>
+              <option value="Marcq en Baroeul">Marcq en Baroeul</option>
+              <option value="Mons en Baroeul">Mons en Baroeul</option>
+              <option value="Hellemmes">Hellemmes</option>
+              <option value="Ronchin">Ronchin</option>
+              <option value="Loos">Loos</option>
+              <option value="Faches-Thumesnil ">Faches-Thumesnil </option>
+            </select>
+          </div>
+          {isLille && (
+            <div className="adress">
+              <h2>Quartier :</h2>
+              <select value={inputDistrict} onChange={handleChangeDistrict}>
+                <option selected value="">
+                  {' '}
```

Comment:
Same here :)

Reply:
Impossible to change with the custom repo